docs/section4/4-10.md

```diff
@@ -0,0 +1,43 @@
+# Airside systems – Rule 4-10
+
+**Rule ID:** 4-10  
+ 
+**Rule Description:** Schedules for HVAC fans that provide outdoor air for ventilation shall be cycled ON and OFF to meet heating and cooling loads during unoccupied hours excluding HVAC systems that meet Table G3.1-4 Schedules for the proposed building exceptions #s 2 and 3.  
+**Rule Assertion:** P-RMR = expected value                                           
+**Appendix G Section:** Table G3.1-4 Schedules for the proposed building excluding exceptions #s 2 and 3  
+**Appendix G Section Reference:** None  
+**Data Lookup:** None  
+**Evaluation Context:** Evaluate each HeatingVentilationAirAconditioningSystem  
+**Applicability Checks:**  
+
+1. Not applicable to HVAC systems in which there are health and safety mandated minimum ventilation requirements during unoccupied hours.
+2. Not applicable to HVAC systems primarily serving computer rooms. 
+ 
+**Function Call:** 
+
+1. get_zones_computer_rooms()
```

Comment:
I commented on this function so this may be returning HVAC systems primarily serving computer rooms instead of list of zones.